plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotSecuredConnectorSmokeTest.java

```diff
@@ -1,45 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.plugin.pinot;
-
-import com.google.common.collect.ImmutableMap;
-import org.junit.jupiter.api.parallel.Isolated;
-
-import java.util.Map;
-
-import static io.trino.plugin.pinot.auth.PinotAuthenticationType.PASSWORD;
-
-@Isolated
-public class TestPinotSecuredConnectorSmokeTest
-        extends BasePinotConnectorSmokeTest
```

Comment:
Running the entire smoke test just for verifying password authentication looks overkill. Also, we should avoid test inheritance as much as possible. https://trino.io/docs/current/develop/tests.html#avoid-test-inheritance